### ISSUE TITLE
Linux: keybinding fails to work: spell-check

### ISSUE THREAD

**Issue by PaulBeaudet** [atom/spell-check#76]
Just spent some time attempting to get the keybinding to trigger for this package in linux. I don't think there is a functioning a user facing way to do it. Maybe I'll look under the hood latter, in the mean time what am I supposed to be expecting when the keybinding works? I would expect the context menu for spell-check to come up. Is that the case? 

The default keybinding is "ctrl-", which I assume might as well be none. I tried Ctrl-shift-N and a few others, but the default keybinding for that combination would not over-ride when I changed it (both spell check and new window) in my keymap file. Is there a user facing way to disable unwanted keybindings. Maybe I can get it to work then.

Atom 1.0.0

**Comment by oraculus** [comment]
+1, can confirm this under Kubuntu 15.04 and Atom 1.0.7
sadly this package seems quite broken :(

**Comment by yagilm** [comment]
+1
perhaps we could have the suggestions in the right click menu?

**Comment by izuzak** [comment]
I don't have access to a Linux machine or VM, so I'd like to ask you @PaulBeaudet @oraculus @yagilm for some help in investigating this.

In the issue, @PaulBeaudet said:

> The default keybinding is "ctrl-", which I assume might as well be none.

But that's not true, the default keybinding on Linux is `ctrl-:` (notice the colon):

https://github.com/atom/spell-check/blob/4fea6982fec0dc733ddebf63c35d298f98a87da5/keymaps/spell-check.cson#L14

Can you please open the [keybinding resolver](https://github.com/atom/keybinding-resolver#keybinding-resolver-package-) and then hit `ctrl-:` within a file which is supported for spellchecking, while the cursor is within a word that has a misspelling:

https://github.com/atom/spell-check#spell-check-package-

Can you provide a GIF which clearly shows the editor (position of cursor within a misspelled word), the grammar which is set for that file (bottom right corner) and the keybinding resolver as you're doing that action? 

For example, a GIF like this (this is from my OSX machine):

![mis](https://cloud.githubusercontent.com/assets/38924/9836566/987ff682-5a1e-11e5-95f0-af9f4aa425cd.gif)

**Comment by yagilm** [comment]
Hello izuzak and thank you for your interest!

I am not sure how to make a gif but the 'key binding resolver' gave me some interesting info.
When I'm trying to <Ctrl-:> on my german keyboard I see < Ctrl-> >, meaning that even though my keyboard layout is German, the keybinding is mapped as an english(?) keyboard.
So to open the Correct spelling menu I need to <Ctrl-Shift-ü> which is quite weird when you think that ':' should be on ö (german hardware keyboard-english layout)..

FYI, I have no keyboard layouts that are mapping ':' on (german ü)

**Comment by oraculus** [comment]
my keyboard distribution (qwerty spanish) brings `:` with `shift-.` but when hitting `ctrl-shift-.` the keybinding resolver shows `ctrl->`

**Comment by izuzak** [comment]
Thanks, @oraculus @yagilm -- that's very helpful. Seems like you're using a non-US keyboard and are hitting a problem with international keyboards (which is not related to this package alone). See https://github.com/atom/atom-keymap/issues/37 (and perhaps https://github.com/atom/atom-keymap/issues/35) for more context. I'm going to close this in favor of that issue so please subscribe there for updates -- and check out those threads, I've seen folks work around some of these issues with keymaps.

Sorry for the trouble with this and let me know if I misudnerstood anything.

**Comment by PaulBeaudet** [comment]
I wasn't using an international keyboard, however I did get this working. Colon is just really odd to use a in a key binding. (must have falsely assumed that's how a keybinding was terminated) Honestly it would be better to display as Control Shift Semicolon as the key binding, if this is going to be keyboard specific. That's a semantic detail though, mostly my bad for not seeing it.

The alternative keybinding I tried, "Ctrl Shift N" didn't work because literally speaking it is impossible ( can't shift a shifted character). It's good thing Atom users are programmers because otherwise they would be super confused by such a semantic detail.

Ultimately now that makes this more of preferential issue than a functional one.

Still the right click context listing is an regularly used paradigm that would make this feature more intuitive to use.
